users/tests/test_models.py

```diff
@@ -120,3 +122,41 @@ def test_find_contributor_is_maintainer(user, library_version):
         first_name=user.first_name, last_name=user.last_name
     )
     assert found_user == user
+
+
+def test_invitation_token_creation(invitation_token):
+    assert invitation_token.expiration_date is not None
+
+
+def test_invitation_token_str_value(invitation_token):
+    expected_str_value = f"{invitation_token.user.get_display_name}'s Invitation Token"
+    assert str(invitation_token) == expected_str_value
+
+
+def test_invitation_token_is_expired_property(invitation_token):
+    # Test not expired case
+    not_expired_date = datetime.datetime.now() + datetime.timedelta(days=1)
+    invitation_token.expiration_date = not_expired_date
+    invitation_token.save()
+
+    assert not invitation_token.is_expired
+
+    # Test expired case
+    expired_date = datetime.datetime.now() - datetime.timedelta(days=1)
+    invitation_token.expiration_date = expired_date
+    invitation_token.save()
+
+    assert invitation_token.is_expired
```

Comment:
Pedantic comment (please ignore it if you find too fine-detail): we are not testing the ">" usage, as in equal dates should still return "not expired". This test is annoying to write because we would need to patch `now`...